administrator/language/de-DE/lib_joomla.ini

```
@@ -740,7 +740,7 @@ JLIB_INSTALLER_FAILED_TO_PURGE_UPDATES="Gespeicherte Updates konnten <strong>nic
 JLIB_INSTALLER_FILE_ERROR_MOVE="Fehler beim Verschieben einer Datei: %s"
 JLIB_INSTALLER_INCORRECT_SEQUENCE="Ein Downgrade von Version %1$s zu %2$s ist <strong>nicht</strong> erlaubt."
 JLIB_INSTALLER_INSTALL="Installieren"
-JLIB_INSTALLER_MINIMUM_JOOMLA="Joomla! erfüllt nicht die minimale Versionsanforderung von %s"
+JLIB_INSTALLER_MINIMUM_JOOMLA="Joomla erfüllt nicht die minimale Versionsanforderung von %s"
```

```suggestion
JLIB_INSTALLER_MINIMUM_JOOMLA="Joomla! erfüllt nicht die minimale Versionsanforderung von %s"
```
see Comment on line R97 file administrator/language/de-DE/com_config.ini